src/index.html

```diff
@@ -7,8 +7,56 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo_link">
+        <img
+        src="images/logo.png"
+        alt="Moyo logo"
+        class="logo_img">
```

Comment:
please fix your indentations
check code style for html in your [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) (13 point)

Reply:
Not fixed